assets/src/edit-story/components/form/color/colorPreview.js

```diff
@@ -317,12 +313,7 @@ function ColorPreview({
             <Transparent />
             <CurrentColor role="status" style={previewStyle} />
           </VisualPreviewInsideButton>
-          <TextualPreview>
-            {isMultiple
-              ? MULTIPLE_DISPLAY_VALUE
-              : previewText ||
-                _x('None', 'No color or gradient selected', 'web-stories')}
```

Comment:
@barklund I've removed this since it doesn't seem to be relevant anymore. LMK if you can think of a case in the editor where this would be needed.